connect_telegram_ext/messages.py

```diff
@@ -19,13 +19,13 @@
 # Copyright (c) 2022, Cloudblue Connect
 # All rights reserved.
 #
-DEFAULT_MESSAGE = 'Obtained request №[{id}]({object_link}) in status {object_status}'
+DEFAULT_MESSAGE = 'Obtained request №<a href={object_link}>{id}</a> in status {object_status}'
```

Comment:
Attention: Request №<a href={object_link}>{id}</a> has been updated and is in status {object_status}. Please check if requires your attention.